### PR TITLE
Upgrade to Ember-CLI 2.15 and package airbrake-js via NPM

### PR DESCRIPTION
**Pull request by thec0keman**
Includes:
- Ember CLI on 2.15
- Drop support for Ember 1.13
- ~Update airbrake-js to latest~
- Package airbrake-js via NPM instead of Bower
- Use Chrome on Travis
- Add yarn

### CODE REVIEW

**Comment by thec0keman** [comment]
Er, something isn't working right... hold off on this for now.

**Comment by thec0keman** [comment]
This should be working now.  I rolled back the upgrade of airbrake-js since later versions were causing build failures on ember-cli. (See [related](https://github.com/ember-cli/ember-cli/issues/6989) [issues](https://github.com/airbrake/airbrake-js/issues/309)).

**Comment by eturino** [comment]
@thec0keman the PR fails because of a linter error here: https://github.com/thec0keman/ember-cli-airbrake/blob/master/app/instance-initializers/ember-cli-airbrake.js#L40
remove the `!!` and this should be ready

**Comment by thec0keman** [comment]
@eturino  should be g2g

**Comment by eturino** [comment]
@bantic @mixonic  merge and release? :)

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ ENV.airbrake = {
 }
 ```
 
-Installing the addon will install the bower component for the airbrake client. This addon uses the airbrake client version **0.5.8**.
+Installing the addon will install the NPM component for the airbrake client. This addon uses the airbrake client version **0.9.8**.
```
Comment (eturino) [inline README.md]: actually I think the docs should say 0.5.9 again

**Comment by thec0keman** [comment]
@eturino  good catch!  fixed.

**Comment by mixonic** [comment]
Sorry this took so long to review, this is a good step.

**Pull request merged**